### PR TITLE
CON-1150 Updated the performance docs page with latest up-to-date figures

### PR DESCRIPTION
**Pull request by filipesoliveira**

### CODE REVIEW

File: docs/docs/performance.md [hunk 1]
```diff
@@ -9,33 +9,41 @@ This means that performance inside an enclave will normally be lower than with a
...
-| mandelbrot    |   Host     |        6.385 | ±        0.132 |  ops/s |
-| nbody         |   Enclave  |        1.205 | ±        0.021 |  ops/s |
-| nbody         |   Host     |        1.279 | ±        0.017 |  ops/s |
-| pidigits      |   Enclave  |        9.941 | ±        0.185 |  ops/s |
-| pidigits      |   Host     |       24.722 | ±        0.301 |  ops/s |
-| spectral_norm |   Enclave  |       11.923 | ±        0.274 |  ops/s |
-| spectral_norm |   Host     |       17.345 | ±        0.930 |  ops/s |
+| Benchmark    | SGX   | Score (ops/s) | Score Delta (%) | Error (ops/s)  | Error (%) |
+|--------------|-------|---------------|-----------------|----------------|-----------|
+| empty        | 	Off	| 24892554.58	| N.A.            | ±7868.489      | ±0.03     |
```
Comment (shamsasari) [inline docs/docs/performance.md:27]: The values seem to be copied from the previous PR. Just to be sure, were these re-run on 1.3?

Comment (filipesoliveira) [reply docs/docs/performance.md:27]: Not yet. It takes some time to run the tests.

Comment (filipesoliveira) [reply docs/docs/performance.md]: The values are now up to date.

**Comment by shamsasari** [comment]
@JohnJoser3 can you please take a look at this.

**Review by JohnJoser3 — APPROVED** [review]

**Pull request merged**